test/test_helper.rb

```diff
@@ -17,6 +17,7 @@
 
 # Configure Rails Envinronment
 ENV["RAILS_ENV"] = "test"
+ENV["VIEW_COMPONENT_ENV"] = "test"
```

Comment:
As an alternative to setting an environment variable, what are your thoughts on adding a method to `ViewComponent::Deprecation` like `.silence_deprecation_warnings!` that we can call early on? That might also give us something to hook into if we wanted to re-enable deprecation warnings for a test later on.

Reply:
Interesting idea. I'm still not sure we're going to go down the `ViewComponent::Deprecation` route but I'll think about it some more 💖